papers.md

```diff
@@ -0,0 +1,207 @@
+
+#### 2. Open an issue and announce the writing of the paper
+
+The Coordinator should open an issue to organize the writing of the paper on
+that repository and then announce to the Jupyter Google Group that the paper is
+being worked on and that anyone can contribute (with a link to the issue).
+
+#### 3. Draft the paper in the repository
+
+At this point, the Coordinator should organize the writing of the actual paper.
+
+The [JOSS Author Guidelines](http://joss.theoj.org/about) detail the format and
+requirements of a JOSS paper. In general this will involve creating a `paper`
+subdirectory in the root of the repository with two files: `paper.md` and
+`paper.bib`. The paper should be written using our standard processes for pull
+requests and code review.
+
+#### 3. Email potential authors
+
+Once a final draft of the paper has been merged into the repository, the
+Coordinator should email all potential authors of the paper, inviting them to
+participate. Usually this will include at least two emails:
+
+* Individually email all contributors listed in the Git log of the repository.
+  This list can be generated using the Git command `git log --all --format='%cN
+  <%cE>' | sort -u`.
+* Email the main Jupyter Google Group to include folks not listed in the Git
+  log.
+
+Both of these emails should include:
+
+* A verbatim copy of the above JOSS authorship criteria;
+* A description the tasks each individual needs complete to be an author on the
+  paper;
+* A concrete deadline for completing those tasks (at least 2 weeks out); and
+* A link to the paper's markdown file in the repository.
+
+The tasks each individual needs to complete are as follows:
+
+* Create an account on [ORCID](http://orcid.org/).
+* Submit a PR to the repository adding their name, ORCID id, and affiliation to
+  the paper;
+* Provide any edits to the content of the paper;
+* In the comment of the PR, briefly describe their contributions to the work;
+  and
+* In the comment of the PR, provide verification that they have read the paper
+  and agree to its publication.
+
+As the deadline approaches, the Coordinator should remind potential authors to
+complete their tasks.
+
+#### 4. Final submission
+
+Once the deadline for authors to complete their tasks has passed, the
+Coordinator should make sure authors are listed in alphabetical order. The
+Coordinator can then submit the paper.
+
+### Process for traditional academic papers
+
+Papers submitted to other (non-JOSS) journals, will usually be longer and take
+a more significant amount of time to author. They will also usually be authored
+in dedicated repository (under the jupyter-resources GitHub organization).
+Because of these factors, the process is slightly different.
+
+### Authorship criteria
+
+The authorship criteria for non-JOSS papers is slightly different than for JOSS
+in that all authors are expected to actively participate in the writing and
+editing of the paper:
```

Comment:
dangling `:`? Doesn't seem to follow into next paragraph as a colon...

This is a minor point, but I'm not sure it will be practical to have *all* authors actively writing say a main Jupyter paper, simply b/c we could have potentially 500+ authors, and I simply can't imagine 500 people wordsmithing together...  E.g. for the Higgs, I don't imagine all 5,000+ authors of the main paper actually edited it, given that (minus references/author list) the paper has ~15,000 words, it would amount to 3 words per person :)

So I think we should say something like:
"""
all authors are expected to actively participate in the editing of the paper. In cases of very large author lists where practicality dictates that a smaller subset does the bulk of the writing, all authors are still expected to read and approve 100% of the final content, contributing edits as necessary.
"""
or similar...